### DISCUSSION TITLE
TLS Client Auth Configuration in Infinity Plugin

### DISCUSSION THREAD

**Discussion by viven1983** [grafana/grafana-infinity-datasource#318] [Q&A]
Hi,

I am configuring the TLS Client Auth Configuration in Infinity Plugin but it is not working when I use the data source and create a Panel to invoke a rest api. It says the below. **Any help on what is going wrong?** I pass the Client Cert (-----BEGIN CERTIFICATE-----
xxxxxx
-----END CERTIFICATE-----) and Client Key (-----BEGIN RSA PRIVATE KEY-----
xxxxx
-----END RSA PRIVATE KEY-----) in the same format as requested. 

{"status":500,"statusText":"Internal Server Error","data":{"message":"Query data error","traceID":"00000000000000000000000000000000"},"config":{"url":"api/ds/query","method":"POST","data":{"queries":[{"columns":[],"datasource":{"uid":"9UWvgH97z","type":"yesoreyeram-infinity-datasource"},"filters":[],"format":"table","global_query_id":"","refId":"A","root_selector":"","source":"url","type":"xml","url":"<<(**https://*.com/x/restapi/invoke/apiname**)>>","url_options":{"data":"<<Input **XML**>>","method":"**POST**"},"datasourceId":3,"intervalMs":30000,"maxDataPoints":639}],"range":{"from":"2022-05-30T06:13:50.895Z","to":"2022-05-30T12:13:50.895Z","raw":{"from":"now-6h","to":"now"}},"from":"1653891230895","to":"1653912830895"},"requestId":"Q100","retry":0,"headers":{"X-Grafana-Org-Id":1},"hideFromInspector":false},"message":"Query data error"}

Thanks,
Viven

**Comment by stphdee** [comment]
I have also the same problem. Have you solved it?

**Reply by hector-baez** [reply]
Did y'all come up with a solution? I'm currently running into the same problem.
